docs/Quickstart/README.md

```diff
@@ -138,4 +138,4 @@ For additional information, see "[Self-Service Operations](https://mariadb.com/d
 
 [Launch DB using the REST API](Quickstart%20da3c5848d30348b1815026b2d24219a6/Launch%20DB%20using%20the%20REST%20API%20cf85a5036a454ee987bb392df694dfc3.md)
 
-[Launch DB using the Terraform Provider](Quickstart%20da3c5848d30348b1815026b2d24219a6/Launch%20DB%20using%20the%20Terraform%20Provider%20ae9f036b2d7c43b3a6e162bcdd4b4c97.md)
```

Comment:
SK looks like you removed the link, but not put a replacement one? Intentionally?